### ISSUE TITLE
Ember Server failing on test-support.js on v1.2.4

### ISSUE THREAD

**Issue by ggayowsky** [Esri/ember-cli-amd#41]
On v1.2.3, `ember server` would successfully complete (though the application wouldn't run).

Now on v1.2.4 `ember server` errors out with the following message

```
Build Error (RequireFilter) in assets/test-support.js

Cannot read property 'length' of undefined

ERROR Summary:

  - broccoliBuilderErrorStack: TypeError: Cannot read property 'length' of undefined
    at new Scanner (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima/dist/esprima.js:5141:29)
    at new Parser (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima/dist/esprima.js:1851:25)
    at parse (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima/dist/esprima.js:120:19)
    at Object.parseScript (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima/dist/esprima.js:145:13)
    at replaceRequireAndDefine (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/ember-cli-amd/index.js:443:23)
    at /home/ggayowsky/Workbench/DIRECT/rspace/node_modules/ember-cli-amd/index.js:499:33
    at walk (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima-walk/esprima-walk.js:12:3)
    at replaceRequireAndDefine (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/ember-cli-amd/index.js:451:3)
    at /home/ggayowsky/Workbench/DIRECT/rspace/node_modules/ember-cli-amd/index.js:499:33
    at walk (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima-walk/esprima-walk.js:12:3)
  - codeFrame: Cannot read property 'length' of undefined
  - errorMessage: assets/test-support.js: Cannot read property 'length' of undefined
        in /tmp/broccoli-1643pIf0FdkyAuGn/out-186-append_ember_auto_import_analyzer
        at RequireFilter
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: assets/test-support.js
    - line: [undefined]
    - treeDir: /tmp/broccoli-1643pIf0FdkyAuGn/out-186-append_ember_auto_import_analyzer
  - message: assets/test-support.js: Cannot read property 'length' of undefined
        in /tmp/broccoli-1643pIf0FdkyAuGn/out-186-append_ember_auto_import_analyzer
        at RequireFilter
  - name: BuildError
  - nodeAnnotation: [undefined]
  - nodeName: RequireFilter
  - originalErrorMessage: Cannot read property 'length' of undefined
  - stack: TypeError: Cannot read property 'length' of undefined
    at new Scanner (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima/dist/esprima.js:5141:29)
    at new Parser (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima/dist/esprima.js:1851:25)
    at parse (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima/dist/esprima.js:120:19)
    at Object.parseScript (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima/dist/esprima.js:145:13)
    at replaceRequireAndDefine (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/ember-cli-amd/index.js:443:23)
    at /home/ggayowsky/Workbench/DIRECT/rspace/node_modules/ember-cli-amd/index.js:499:33
    at walk (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima-walk/esprima-walk.js:12:3)
    at replaceRequireAndDefine (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/ember-cli-amd/index.js:451:3)
    at /home/ggayowsky/Workbench/DIRECT/rspace/node_modules/ember-cli-amd/index.js:499:33
    at walk (/home/ggayowsky/Workbench/DIRECT/rspace/node_modules/esprima-walk/esprima-walk.js:12:3)

=================================================================================
```

Here is my projects package.json dependencies

  "devDependencies": {
    "@ember/jquery": "^0.5.2",
    "@ember/optional-features": "^0.6.3",
    "broccoli-asset-rev": "^2.7.0",
    "ember-auto-import": "^1.2.10",
    "ember-cli": "~3.7.1",
    "ember-cli-amd": "1.2.4",
    "ember-cli-app-version": "^3.2.0",
    "ember-cli-babel": "^7.1.2",
    "ember-cli-cjs-transform": "^1.3.0",
    "ember-cli-content-security-policy": "^1.0.0",
    "ember-cli-dependency-checker": "^3.0.0",
    "ember-cli-eslint": "^4.2.3",
    "ember-cli-htmlbars": "^3.0.0",
    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
    "ember-cli-inject-live-reload": "^1.8.2",
    "ember-cli-sass": "^8.0.1",
    "ember-cli-sri": "^2.1.1",
    "ember-cli-uglify": "^2.1.0",
    "ember-component-css": "^0.6.5",
    "ember-concurrency": "^0.8.26",
    "ember-export-application-global": "^2.0.0",
    "ember-fetch": "^5.1.3",
    "ember-load-initializers": "^1.1.0",
    "ember-qunit": "^3.4.1",
    "ember-resolver": "^5.0.1",
    "ember-source": "~3.7.0",
    "ember-truth-helpers": "^2.0.0",
    "eslint": "^5.7.0",
    "eslint-config-semistandard": "^12.0.1",
    "eslint-config-standard": "^11.0.0",
    "eslint-plugin-ember": "^5.2.0",
    "eslint-plugin-header": "^1.2.0",
    "eslint-plugin-import": "^2.8.0",
    "eslint-plugin-node": "^7.0.1",
    "eslint-plugin-promise": "^4.0.0",
    "eslint-plugin-standard": "^3.0.1",
    "glob": "^4.5.3",
    "http-proxy": "^1.17.0",
    "loader.js": "^4.7.0",
    "morgan": "^1.9.0",
    "sass": "^1.14.3",
    "sinon": "^7.0.0"
  },
  "engines": {
    "node": "6.* || 8.* || >= 10.*"
  },
  "dependencies": {
    "d3": "^5.7.0",
    "d3-scale-chromatic": "^1.3.3",
    "font-awesome": "^4.7.0",
    "jquery-ui-dist": "^1.12.1",
    "jquery-ui-touch-punch": "^0.2.3",
    "moment": "^2.22.2",
    "normalize.css": "^3.0.3",
    "nouislider": "^11.1.0",
    "spin.js": "^3.1.0",
    "tinycolor2": "^1.4.1"
  }

**Comment by ggayowsky** [comment]
I am doing some more debugging now, however, this also seems to be related to `ember-auto-import`

**Comment by ffaubry** [comment]
Fix released in 2.0.2
